src/azure-cli/azure/cli/command_modules/appservice/_constants.py

```diff
@@ -39,39 +39,18 @@
         'java': '8'
     }
 }
-# functions version -> runtime -> runtime version : container image
-RUNTIME_TO_IMAGE_FUNCTIONAPP = {
+# functions version -> runtime : runtime versions
+RUNTIME_TO_SUPPORTED_VERSIONS_FUNCTIONAPP = {
     '2': {
-        'node': {
-            '8': 'mcr.microsoft.com/azure-functions/node:2.0-node8-appservice',
```

Comment:
Will the values like 'mcr.microsoft.com/azure-functions/node:2.0-node8-appservice' not be used any more?

Reply:
Correct -- the back end has been updated to know which container image to use, so it's not something we have to set here anymore